### PR TITLE
Simplifying ->app

### PR DESCRIPTION
**Pull request by ianffcs**

### CODE REVIEW

File: src/framework/components/app/core.clj [hunk 1]
```diff
@@ -117,34 +117,52 @@
...
+          #(if config
+             (acl-builder/init % config)
+             %)))
+
 (defn ->app
-  [{acl-cfg         :acl-cfg
+  [{_acl-cfg        :acl-cfg
     session-backend :session-backend
-    auth            :auth
+    _auth           :auth
```
Comment (LeaveNhA) [inline src/framework/components/app/core.clj:136]: Why underscoring? Implies the localization of the binding?

Comment (LeaveNhA) [reply src/framework/components/app/core.clj:136]: And I think they are not used. Or forgotten?

Comment (ianffcs) [reply src/framework/components/app/core.clj:136]: They were not used in past versions, like they were some type of a component, but now you can pass them when you need to use, just underscoring to not forget them and to linter not complain about it.

Comment (g-krisztian) [reply src/framework/components/app/core.clj:136]: Just please do not remove functionality from framework. Make it possible to turn it on-off in the app. (I believe it's enough to not pass the value in the map to turn it off)

**Review by g-krisztian — CHANGES_REQUESTED** [review]

**Review by g-krisztian — APPROVED** [review]

**Review by narocath — APPROVED** [review]

**Pull request merged**